db/seeds.rb

```diff
@@ -8,7 +16,7 @@
       email_address: hash.fetch('email', nil),
       enabled: hash.fetch('enabled', true),
       estate: hash.fetch('estate'),
-      id: hash.fetch('id'),
+      id: prison_uuid_mappings[path],
```

Comment:
Which is harder than you'd expect with rubocop, which dislikes put statements in `db/seeds.rb`.  As I want it to be very clear what is going on, I'm using a `fail` instead.